src/compile/common.ts

```diff
@@ -78,7 +78,8 @@ export function formatSignalRef(
   fieldDef: TypedFieldDef<string>,
   specifiedFormat: string,
   expr: 'datum' | 'parent' | 'datum.datum',
-  config: Config
+  config: Config,
+  coerce = true
```

Comment:
boolean param is bad.   Reading the call like `text(fieldDef, config, expr, false)` gives us no idea what is false for at all. 

Also, coerce is also vague. `castToString`? or something similar?


Like `text(fieldDef, config, expr, {castToString: true/false})` ?

Reply:
I'm happy to rename the property but expecting an object instead of a boolean is weird. In e.g. `timeFormatExpression`, we also don't use this pattern.

Reply:
I guess we started doing this in a few other places so I will update this.